Metadata.yaml

```diff
@@ -0,0 +1,245 @@
+        GitHub: nilsdeppe
+      - Name: Throwe, William
+        Orcid: 0000-0001-5059-4378
+        Affiliations: [*cornell]
+        GitHub: wthrowe
+      - Name: Kidder, Lawrence E.
+        Orcid: 0000-0001-5392-7342
+        Affiliations: [*cornell]
+        GitHub: kidder
+      - Name: Fischer, Nils L.
+        Orcid: 0000-0002-5767-3949
+        Affiliations: [*aei]
+        GitHub: nilsleiffischer
+
+  Developers:
+    Description: |
+      Developers have contributed major features to SpECTRE and remain in this
+      list indefinitely (unless they wish to be removed).
+
+      Guidelines on becoming a Developer:
+
+      - Do some code review, does not need to be self-initiated.
+      - Somewhere around 40 commits and 10k-15k lines of code should trigger a
+        discussion among the Core developers to move someone from Contributors
+        to Developers. These are guidelines and being above or below these
+        arbitrary cutoffs does not automatically include or exclude you. E.g.
+        someone doing complicated low-level code will likely become a Developer
+        with far fewer lines of code contributed.
+
+      This author list is ordered by last name.
+    List:
+      - Name: Armaza, Cristóbal
+        Orcid: 0000-0002-1791-0743
+        Affiliations: [*cornell]
+        GitHub: carmaza
+      - Name: Bonilla, Gabriel S.
+        Orcid: 0000-0003-4502-528X
+        Affiliations: [*cornell]
+        GitHub: gsb76
+      - Name: Hébert, François
+        Orcid: 0000-0001-9009-6955
+        Affiliations: [*caltech]
+        GitHub: fmahebert
+      - Name: Kumar, Prayush
+        Orcid: 0000-0001-5523-4603
+        Affiliations: [*cornell]
+        GitHub: prayush
+      - Name: Lovelace, Geoffrey
+        Orcid: 0000-0002-7084-1070
+        Affiliations: [*fullerton]
+        GitHub: geoffrey4444
+      - Name: Moxon, Jordan
+        Orcid: 0000-0001-9891-8677
+        Affiliations: [*caltech]
+        GitHub: moxcodes
+      - Name: O'Shea, Eamonn
+        Affiliations: [*cornell]
+        GitHub: osheamonn
+      - Name: Scheel, Mark A.
+        Orcid: 0000-0001-6656-9134
+        Affiliations: [*caltech]
+        GitHub: markscheel
+      - Name: Teukolsky, Saul A.
+        Orcid: 0000-0001-9765-4526
+        Affiliations: [*caltech, *cornell]
+        GitHub: teukolsky
+
+  Contributors:
+    Description: |
+      Anyone who has contributed to SpECTRE will be added to this list and
+      remain a Contributor indefinitely (unless they wish to be removed).
+
+      Guidelines on becoming a Contributor:
+
+      - See our guidelines in `docs/CONTRIBUTING.md` for information on how to
+        get involved with SpECTRE development.
+      - Having any PR merged qualifies for this list. If multiple people worked
+        on the PR together, e.g. several undergraduate students contributing to
+        a single PR, all persons involved will become a Contributor.
+
+      This author list is ordered by last name.
+
```

Comment:
[nitpick] in the above Developer and Core parts you do not have a blank line here